systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -295,6 +296,7 @@ void testKafkaConnectAndConnectorMetrics(ExtensionContext extensionContext) {
     @IsolatedTest
     @Tag(INTERNAL_CLIENTS_USED)
     void testKafkaExporterMetrics(ExtensionContext extensionContext) {
+        TestStorage testStorage = storageMap.get(extensionContext);
```

Comment:
```suggestion
        final TestStorage testStorage = storageMap.get(extensionContext);
```
and in other cases please update it :)